Dockerfile.cosign.rh

```diff
@@ -1,5 +1,5 @@
 # Build stage
```

Comment:
use newer version `sha256:44fd8f88f3b6463cda15571260f9ca3a0b78d3c8c8827a338e04ab3a23581a88`, I hope that it will solve problem with discovered CVE by EC